src/measure_theory/function/conditional_expectation.lean

```diff
@@ -2336,6 +2380,56 @@ end
 
 end indicator
 
+section real
+
+/-- Given a integrable function `g`, the conditional expectations of `g` with respect to
+a sequence of sub-σ-algebras is uniformly integrable. -/
+lemma mem_ℒp.uniform_integrable_condexp {ι : Type*} [is_finite_measure μ]
```

Comment:
We usually use `integrable g μ` instead of `mem_ℒp g 1 μ`. Could you change this lemma and `mem_ℒp.uniform_integrable_condexp_filtration` to use integrable?